### PR TITLE
Add 1.1.11 and 1.1.12 to CHANGELOG.md

### PR DESCRIPTION
**Pull request by remram44**
# Pull Request Check List

- `n/a` Added **tests** for changed code.
- `n/a` Updated **documentation** for changed code.
- [x] This PR changes no code

I took the info from the GitHub releases page. I couldn't find any info for 1.1.13, so I didn't add it.

See also #4554

### CODE REVIEW

**Review by finswimmer — CHANGES_REQUESTED** [review]
Hello @remram44,

thanks for porting this data. I've added the 1.1.13 info to the release page as well. You could find all these data in the `CHANGELOG.md` of the 1.1 branch as well: https://github.com/python-poetry/poetry/blob/1.1/CHANGELOG.md

fin swimmer

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
 # Change Log
 
+## [1.1.13] - 2022-02-09
+
+## Fixed
+
+- Fixed an issue where envs in MSYS2 always reported as broken ([#4942](9https://github.com/python-poetry/poetry/pull/4942))
```
Comment (finswimmer) [inline CHANGELOG.md:7]:
```suggestion
- Fixed an issue where envs in MSYS2 always reported as broken ([#4942](https://github.com/python-poetry/poetry/pull/4942))
```

**Review by finswimmer — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (remram44) [reply CHANGELOG.md]: Error exists on `1.1` branch too

**Review by finswimmer — APPROVED** [review]
Thanks a lot :+1:

**Comment by tony** [comment]
@finswimmer minor fix for the heading (i didn't catch this initially) #5237

**Pull request merged**